elementary/monitor/dbt_project/macros/upload_source_freshness.sql

```diff
@@ -1,5 +1,5 @@
 {% macro upload_source_freshness(results) %}
-  {% set source_freshness_results_dicts = fromjson(results) %}
+  {% set source_freshness_results_dict = fromjson(results) %}
```

Comment:
I think this does take multiple dicts though as `target/sources.json:results` is a list.
Otherwise it wouldn't work since `upload_artifacts_to_table` takes a list.

Reply:
renaming to `source_freshness_results` :)